cmd/influxd/run/server_suite_test.go

```diff
@@ -19,13 +19,13 @@ func init() {
 			&Query{
 				name:    "create database should succeed",
 				command: `CREATE DATABASE db0`,
-				exp:     `{"results":[{}]}`,
+				exp:     `{"results":[{"statement_id":0}]}`,
```

Comment:
Statement ID's would have to start at `1`, not `0` then, as `0` can be a valid statement ID it looks like.  I would agree that I would like to see it omitted as it creates a lot of noise when there is no response to be had.